app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,27 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            if (Desktop.isDesktopSupported()) {
+                Desktop.getDesktop().browse(new URI("https://github.com/CIRDLES/Topsoil/wiki"));
```

Comment:
Check the documentation again. Just because the desktop is supported doesn't mean that accessing the browser is.